### PR TITLE
Update readme + composer requirements; move license file and drop bui…

### PR DESCRIPTION
**Pull request by usox**
I updated the readme and the composer.json according to #41. Also, I removed the build.xml and the phing targets from the Makefile due to dependency problems to PEAR-Packages (FileManager at least). The only used phing target was `tar` which (from what I understand) is/was used to build a release package. Any thoughts about that?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,64 +1,55 @@
 
 # PHPTAL - Template Attribute Language for PHP
 
-Master: [![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
+[![Monthly Downloads](https://poser.pugx.org/phptal/phptal/d/monthly)](https://packagist.org/packages/phptal/phptal)
+[![License](https://poser.pugx.org/phptal/phptal/license)](https://packagist.org/packages/phptal/phptal)
```
Comment (Ocramius) [inline README.md]: should be `LICENSE`

File: README.md [hunk 2]
```diff
@@ -1,64 +1,55 @@
...
-============
-
-Get the latest PHPTAL package from http://phptal.org.
-
-Then run:
-
-    pear install PHPTAL-X.X.X.tar.gz
-
-
+[Please see the projects releases page](https://github.com/phptal/PHPTAL/releases)
```
Comment (Ocramius) [inline README.md]: Note: need to makes sure we make these notes available as part of the repo as well:
 - as a file?
 - as the tag message (`git tag -F <some-release-notes>`)

This needs to be added to some contributing notes

File: README.md [hunk 3]
```diff
@@ -1,64 +1,55 @@
...
 
-  - PHPUnit 3.4 to run tests
+Addition development requirements (optional)
+============================================
 
-        pear channel-discover pear.phpunit.de
-		pear channel-discover pear.symfony-project.com
-		pear channel-discover components.ez.no
-        pear install phpunit/PHPUnit
+If you would like to generate the offical html/text handbook, you will need to install
```
Comment (Ocramius) [inline README.md]: Maybe add the command used to generate docs?

File: composer.json [hunk 4]
```diff
@@ -1,39 +1,41 @@
...
-        "bin-dir": "tools"
-    }
+	"name": "phptal/phptal",
+	"homepage": "http://phptal.org/",
+	"description": "PHPTAL is a templating engine for PHP5 that implements Zope Page Templates syntax",
+	"version":"1.3.0",
+	"license": "LGPL-2.1+",
+	"type": "library",
+	"keywords": [
+		"zope",
```
Comment (Ocramius) [inline composer.json]: `s/\t/    /` please

File: tests/run-tests.php [hunk 5]
```diff
@@ -13,8 +13,6 @@
  * @link     http://phptal.org/
  */
 
-require_once 'PHPUnit/Autoload.php';
```
Comment (Ocramius) [inline tests/run-tests.php]: Is this file even needed?

File: README.md [hunk 6]
```diff
@@ -1,64 +1,55 @@
 
 # PHPTAL - Template Attribute Language for PHP
 
-Master: [![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
+[![Monthly Downloads](https://poser.pugx.org/phptal/phptal/d/monthly)](https://packagist.org/packages/phptal/phptal)
+[![License](https://poser.pugx.org/phptal/phptal/license)](LICENSE)
+[![Build Status](https://secure.travis-ci.org/pornel/PHPTAL.png?branch=master)](http://travis-ci.org/pornel/PHPTAL)
```
Comment (potherca) [inline README.md]: The link to Pornel's travis is broken.. I've set up a new one: https://travis-ci.org/phptal/PHPTAL
New code is:
 `[Build Status](https://travis-ci.org/phptal/PHPTAL)`

File: README.md [hunk 7]
```diff
@@ -1,64 +1,55 @@
...
 
-Non-PEAR install
-================
+```
+{
+    "require": {
+        "phptal/phptal": "~1.3"
+    }
+}
+```
```
Comment (potherca) [inline README.md]: Editing the `composer.json` file is discouraged. Use `composer require phptal/phptal` instead.

File: README.md (hunk 2 above)
Comment (potherca) [inline README.md]: I'll put that in an issue for the v2 milestone.

**Comment by potherca** [comment]
First of all: thank you for the work thus far @usox !

Other than a few minor pointers this looks good to me. I feel inclined to merge this. 

@Ocramius / @Pornel ?

(Also, do we want this to go on master or a v2 branch and integrate later? I think we need to start making that `CONTRIBUTING ` file...)

**Comment by Ocramius** [comment]
@usox @potherca merged and created branch `1.3.x` off of tag `1.3.0`.

`master` is `2.x` if you want (we can alias it in a different PR if you prefer so)

**Comment by Ocramius** [comment]
Also applied the review comments

**Pull request merged**